Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/attendee/checkins/CheckInsFragment.kt

```diff
@@ -53,7 +53,8 @@ class CheckInsFragment : Fragment(R.layout.trace_location_attendee_checkins_frag
             factory as CheckInsViewModel.Factory
             factory.create(
                 savedState = savedState,
-                deepLink = navArgs.uri
+                deepLink = navArgs.uri,
+                cleanHistory = navArgs.cleanHistory
```

Comment:
I think it is better if you call it `clearHistory`

Reply:
I have nothing against using `clear` vs `clean` it could be refactored later